{{ cookiecutter.project_shortname }}/setup.cfg

```diff
@@ -1,4 +1,57 @@
-{% include 'misc/header.py' %}
+{% include "misc/header.py" %}
+
+[metadata]
+name = {{ cookiecutter.project_shortname }}
+version = attr: {{ cookiecutter.package_name }}.__version__
+description = "{{ cookiecutter.description }}"
+long_description = file: README.rst, CHANGES.rst
+keywords = invenio TODO
+license = MIT
+author = {{ cookiecutter.author_name }}
+author_email = {{ cookiecutter.author_email }}
+platforms = any
+url = https://github.com/{{ cookiecutter.github_repo }}
+classifiers =
+    Development Status :: 1 - Planning
+
+[options]
+include_package_data = True
+packages = find:
+python_requires = >=3.7
+zip_safe = False
+install_requires =
+    invenio-base>=1.2.5
+    invenio-i18n>=1.3.1
+
+[options.extras_require]
+tests =
+    invenio-app>=1.3.4,<2.0.0
+    pytest-invenio>=2.1.0,<3.0.0
+    pytest-black>=0.3.0
```

Comment:
minor/shelvable: should this be moved to pytest-invenio since now all modules support black?

Reply:
Same thought: https://github.com/inveniosoftware/cookiecutter-invenio-module/pull/147#discussion_r1101239515

I can open an issue in `pytest-invenio` and we can address it there.

Reply:
Yes! Sorry, it was closed/resolved and I missed it